src/events.ts

```diff
@@ -37,7 +37,7 @@ export interface CommonAccessCodeEvent<
     EventType,
     {
       access_code_id: string
-      code: string
+      code: string | null
```

Comment:
in some scenarios we dont know the code?

Reply:
We will make sure it is instead on this change. 😄